### ISSUE TITLE
Meaningful profiles on social media / open source sites

### ISSUE THREAD

**Issue by davidbeckingsale** [LLNL/open-source-guidelines#3]
Since these are only "guidelines" I think we should recommend (**strongly**) that people populate their profiles on social media and sites like GitHub.

It looks really bad when someone is 'blogs22' with no profile picture or description.

**Comment by IanLee1521** [comment]
I completely agree, certainly at a minimum for those in the @LLNL org should have their profiles filled out. I'd added some documentation to that effect here: https://github.com/LLNL/llnl.github.io/blob/master/about/using-github.md#setting-up-your-github-account

Though based it's still been a bit limited in it's adoption. See the numbers for those with 2FA disabled still: https://github.com/orgs/LLNL/people?utf8=✓&query=%20two-factor%3Adisabled
